cdap-docs/admin-manual/source/troubleshooting.rst

```diff
@@ -2,13 +2,29 @@
     :author: Cask Data, Inc.
     :copyright: Copyright © 2014 Cask Data, Inc.
 
-============================================
-Troubleshooting CDAP
-============================================
+.. _admin-troubleshooting:
```

Comment:
Just a general comment, how does this troubleshooting page relate to the FAQ?  it looks like there is a lot of overlap, and in some cases similar questions with different answers.

For example:
   * From http://builds.cask.co/artifact/CDAP-DOB48/shared/build-1/Docs-HTML/3.2.1/en/admin-manual/troubleshooting.html, 
      * "CDAP services on distributed CDAP don’t start up due to java.lang.ClassNotFoundException"
      * "YARN Application Shows ACCEPTED For Some Time But Then Fails"
   * http://builds.cask.co/artifact/CDAP-DOB48/shared/build-1/Docs-HTML/3.2.1/en/faqs/cdap.html#i-don-t-see-the-cdap-master-service-on-yarn 

are all variations of the same question but with different answers.